frontend/cypress/integration/talentsearch/profile-workflows.spec.js

```diff
@@ -0,0 +1,54 @@
+import { aliasMutation, aliasQuery } from "../../support/graphql-test-utils";
+
+describe("User Profile Smoke Tests", () => {
+
+  const loginAndGoToProfile = () => {
+    cy.login("applicant");
+    cy.visit("/en/talent/profile");
+
+    // make sure we end up on the profile page
+    cy.wait("@gqlgetMeQuery");
+    cy.findByRole("heading", { name: /My Status/i })
+      .should("exist")
+      .and("be.visible");
+    cy.url().should("contain", "/profile");
+  };
+
+  beforeEach(() => {
+    cy.intercept("POST", "/graphql", (req) => {
+      aliasQuery(req, "getMe");
+      aliasMutation(req, "UpdateUserAsUser");
+      aliasMutation(req, "createWorkLocationPreference");
+    });
+
+    loginAndGoToProfile();
+  });
+
+  it("Reviews a user profile and makes some edits", () => {
+    // about me
+    cy.findByRole("link", { name: /Edit About Me/i }).click();
+    cy.findByRole("textbox", { name: /Current city/i })
+      .clear()
+      .type("Test City");
+    cy.findByRole("button", { name: /Save and go back/i }).click();
+    cy.wait("@gqlUpdateUserAsUserMutation");
+    cy.findByRole("alert")
+      .findByText(/User updated successfully/i)
+      .should("exist")
+      .and("be.visible");
```

Comment:
this can be a command as well..probably verifyUserUpdatedSuccessfully()

Reply:
This set of commands is so small that I wonder whether its worth it but it will be used throughout the app so I added it anyway.  https://github.com/GCTC-NTGC/gc-digital-talent/pull/3084/commits/5a8c25ca861dea017d60e06585c55a19324e102c